libr/core/cmd_anal.c

```diff
@@ -4244,7 +4244,7 @@ static void cmd_anal_syscall(RCore *core, const char *input) {
 	case 'c': // "asc"
 		if (input[1] == 'a') {
 			if (input[2] == ' ') {
-				if ((n = atoi (input + 2)) > 0) {
+				if ((n = atoi (input + 2)) >= 0) {
```

Comment:
thats wrong. because atoi("write") will result in syscall 0 which is wrong and should go to the else branch